File: Tienda_Java/src/Persona.java
public class Persona {
    String nombre;
    String direccion;

    Persona(String nombre, String direccion) {
        this.nombre = nombre;
        this.direccion = direccion;
    }

    String getNombre() {
        return nombre;
    }

    void setNombre(String nombre) {
        this.nombre = nombre;
    }

    String getDireccion() {
        return direccion;
    }

    void setDireccion(String direccion) {
        this.direccion = direccion;
    }

}

File: Tienda_PY/Persona.py
class Persona:
    def __init__(self, nombre, direccion):
        self.nombre = nombre
        self.direccion = direccion

    @property
    def nombre(self):
        return self._nombre

    @nombre.setter
    def nombre(self, nombre):
        self._nombre = nombre

    @property
    def direccion(self):
        return self._direccion

    @direccion.setter
    def direccion(self, direccion):
        self._direccion = direccion